app/workers/sentry_worker.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SentryWorker
```

Comment:
Propsoe to create an `ApplicationWorker` and add `include Sidekiq::Worker` to it

Reply:
Method `perform` unavailible when name is `ApplicationWorker` 
I have named `BaseWorker`